### PR TITLE
migrate examples for sum se_resnext and 3d-unet

### PR DESCRIPTION
**Pull request by violetch24**
## Type of Change

migrate examples to INC new API for NLP/summarization, image_recognition/se_resnext, 3d-unet
bug fix for torchvision_models

## Description

JIRA ticket: https://jira.devtools.intel.com/browse/ILITV-2625

### CODE REVIEW

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4413/artifact/report.html)

File: examples/README.md [hunk 1]
```diff
@@ -391,7 +391,11 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>ResNet18</td>
     <td>Image Recognition</td>
     <td>Quantization-Aware Training</td>
+<<<<<<< HEAD
```
Comment (chensuyue) [inline examples/README.md:394]: fix the conflict.

File: examples/.config/model_params_pytorch.json [hunk 2]
```diff
@@ -161,20 +101,16 @@
     "resnet101_qat": {
       "model_src_dir": "image_recognition/torchvision_models/quantization/qat/eager",
```
Comment (chensuyue) [inline examples/.config/model_params_pytorch.json:102]: path issue "image_recognition/torchvision_models/quantization/qat/fx"

File: examples/.config/model_params_pytorch.json [hunk 3]
```diff
@@ -161,20 +101,16 @@
...
       "dataset_location": "/tf_dataset/pytorch/ImageNet/raw",
-      "input_model": "",
-      "yaml": "conf.yaml",
-      "strategy": "basic",
-      "batch_size": 100,
-      "new_benchmark": false
+      "input_model": "resnext101_32x8d",
```
Comment (chensuyue) [inline examples/.config/model_params_pytorch.json:104]: `resnet101` or `resnext101` ？

Comment (violetch24) [reply examples/.config/model_params_pytorch.json:104]: the input model should be `resnext101_32x8d`, so edited the topology name.

File: examples/README.md (hunk 1 above)
Comment (violetch24) [reply examples/README.md]: fixed

File: examples/.config/model_params_pytorch.json (hunk 2 above)
Comment (violetch24) [reply examples/.config/model_params_pytorch.json]: fixed

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4470/)

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4683/)

File: examples/pytorch/image_recognition/3d-unet/quantization/ptq/fx/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  pip install absl-py
+  python setup.py install
+  cd ../..
+```
+
+# Run
+## 1. Quantization
+
+```shell
+  make run_pytorch_NC_tuning
```
Comment (chensuyue) [inline examples/pytorch/image_recognition/3d-unet/quantization/ptq/fx/README.md:49]: Have you local test with this cmd?

File: examples/pytorch/image_recognition/3d-unet/quantization/ptq/fx/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+```
+## 2. Benchmark
+```bash
+# int8
+sh run_benchmark.sh --int8=true --input_model=build/result/nnUNet/3d_fullres/Task043_BraTS2019/nnUNetTrainerV2__nnUNetPlansv2.mlperf.1 --dataset_location=build/preprocessed_data/
+# fp32
+sh run_benchmark.sh --input_model=build/result/nnUNet/3d_fullres/Task043_BraTS2019/nnUNetTrainerV2__nnUNetPlansv2.mlperf.1 --dataset_location=build/preprocessed_data/
+```
+## 3. Model Baseline
+| model | framework | accuracy | dataset | model link | model source | precision | notes |
```
Comment (chensuyue) [inline examples/pytorch/image_recognition/3d-unet/quantization/ptq/fx/README.md:65]: Remove the last empty column.

File: examples/pytorch/nlp/huggingface_models/summarization/quantization/ptq_dynamic/fx/README.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+tar -xzvf billsum.tar.gz
+```
+
+# Run
+## 1. Quantization
+```shell
+cd examples/pytorch/nlp/huggingface_models/summarization/quantization/ptq_dynamic/fx
+sh run_tuning.sh --topology=topology_name
+```
+> NOTE
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/summarization/quantization/ptq_dynamic/fx/README.md:31]: remove the note, already has a table.

File: examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+Step-by-Step
+============
+
+This document describes the step-by-step instructions for reproducing PyTorch se_resnext tuning and benchmarking results with Intel® Neural Compressor.
+
+> **Note**
```
Comment (chensuyue) [inline examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md:6]: same as before.

File: examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+## 1. Environment
+Python 3.6 or higher version is recommended.
+The dependent packages are all in requirements, please install as following.
+```shell
+cd examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx
+pip install -r requirements.txt
+```
+## 2. Install Model
+```shell
+python setup.py install
```
Comment (chensuyue) [inline examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md:22]: Can we also use git clone rather than ship all the code? So we only need the main scripts and readme.

File: examples/pytorch/image_recognition/3d-unet/quantization/ptq/fx/README.md (hunk 4 above)
Comment (violetch24) [reply examples/pytorch/image_recognition/3d-unet/quantization/ptq/fx/README.md:49]: yes, either cmd works.

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4683/)

File: examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md (hunk 8 above)
Comment (violetch24) [reply examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md:19]: done

**Comment by violetch24** [comment]
**topologies to be tested**: pegasus_samsum, 3dunet, resnest50, se_resnext50_32x4d

File: examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+## 2. Benchmark
+```bash
+# int8
+sh run_benchmark.sh --int8=true --config=saved_results --mode=performance --input_model=se_resnext50_32x4d  --dataset_location=/path/to/imagenet
+# fp32
+sh run_benchmark.sh --mode=performance --input_model=se_resnext50_32x4d  --dataset_location=/path/to/imagenet
+```
+
+# Original SE_ResNext README
+Please refer [SE_ResNext README](SE_ResNext_README.md)
```
Comment (chensuyue) [inline examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md:51]: We can delete this readme in our example path and link to an online readme.

**Comment by violetch24** [comment]
**topologies to be tested**: pegasus_samsum, 3dunet, se_resnext50_32x4d

**Review by chensuyue — APPROVED** [review]
